.github/workflows/release.yaml

```
@@ -112,9 +105,9 @@ jobs:
 
             - name: Publish to Google Artifact Registry
               if: steps.unpublish.outputs.already_released != 'true'
-              run: yarn publish --tag ${{ inputs.release_tag }}
+              run: yarn publish --non-interactive --registry=https://europe-west3-npm.pkg.dev/hoprassociation/npm/ --tag release-candidate
               env:
-                  NODE_AUTH_TOKEN: ${{ steps.gcp.outputs.access_token }}
+                NODE_AUTH_TOKEN: ${{ steps.gcp.outputs.access_token }}
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 18 but found 16

(indentation)